lib/exception_notifier.rb

```diff
@@ -96,10 +104,19 @@ def notifiers
     #   ExceptionNotifier.ignore_if do |exception, options|
     #     not Rails.env.production?
     #   end
+    #
+    # It also accepts a condition that is specific to each individual notifier.
```

Comment:
This comment no longer applies right? Since we have `ignore_notifier_if` now

Reply:
My bad... thanks for catching this!